agent/bench-scripts/pbench-fio

```diff
@@ -720,6 +721,18 @@ function fio_run_job() {
 		fi
 	else
 		/bin/mv $benchmark_results_dir/*.log $benchmark_results_dir/clients/localhost/ 2> /dev/null
+		# compute localhost latency percentiles
+		if grep -q "^log_hist_msec" $fio_job_file; then
+			does_wildcard_exist $benchmark_results_dir/clients/localhost/'fio_clat_hist.*.log*'
+			if [[ $? -eq 0 ]]; then
+				mkdir -p $benchmark_results_dir/clients/localhost/hist
+				# compute per-client latency percentiles
+				$histogram_cmd $benchmark_results_dir/clients/localhost/fio_clat_hist.*.log* >$benchmark_results_dir/clients/localhost/hist/hist.csv
+				$script_path/postprocess/$benchmark-postprocess-viz.py --job-file=$fio_job_file ${histogram_params} -- $benchmark_results_dir/clients/localhost/hist
```

Comment:
There seem to be unnecessary differences from the clients code above. Specifically, why put the results into `clients/localhost/hist` instead of `hist`? Without that difference it seems that this post-processing step would be identical with or without `clients` and could just be factored out of the `if/else`.

Reply:
Yeah, it's ugly.  I'll attempt a refactor to unify this behavior.